packages/core/src/components/dropdown/test/error/dropdown.e2e.ts

```diff
@@ -6,79 +6,34 @@ const componentTestPath = 'src/components/dropdown/test/error/index.html';
-    const helperText = dropdown.locator('div[class="helper error"]');
-    expect(helperText).toBeTruthy();
-    const helperErrorIcon = helperText.locator('tds-icon[name="error"]');
-    expect(helperErrorIcon).toBeTruthy();
-  });
+    const dropdown = page.getByTestId('tds-dropdown-testid');
+    await expect(dropdown).toHaveCount(1);
 
-  test('the placeholder="Placeholder" text', async ({ page }) => {
-    await page.goto(componentTestPath);
-    const dropdown = page.locator('tds-dropdown');
-    const button = dropdown.locator('button');
-    const placeholder = button.locator('div[class="placeholder lg"]');
-    const textAttribute = await placeholder.textContent();
-    expect(textAttribute).toBe('Placeholder');
+    /* Check diff on screenshot */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
   });
 
-  test('has lg size', async ({ page }) => {
+  test('find helper text and check if visible and have icon', async ({ page }) => {
     await page.goto(componentTestPath);
-    const dropdown = page.locator('tds-dropdown');
-    const dropdownSelect = dropdown.locator('div[class="dropdown-select lg"]');
-    expect(dropdownSelect).toBeTruthy();
+    const helperText = page.getByText(/Helper text/);
+    await expect(helperText).toBeVisible();
+    const helperTextIcon = helperText.locator('tds-icon');
+    await expect(helperTextIcon).toHaveCount(1);
   });
 
   test('clicking the dropdown opens the dropdown-list', async ({ page }) => {
     await page.goto(componentTestPath);
-    const dropdown = page.locator('tds-dropdown');
-    const dropdownSelect = dropdown.locator('div[class="dropdown-select lg"]');
-    const button = dropdownSelect.locator('button');
-    const myEventSpy = await page.spyOnEvent('click');
-    await button.click();
-    const dropdownList = dropdown.locator('div[class="dropdown-list lg down label-outside open"]');
-    expect(myEventSpy).toHaveReceivedEvent();
-    expect(dropdownList).toBeTruthy();
-    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
-  });
+    const dropdownOptionOne = page.getByTestId(/tds-dropdown-option-1/);
+    const placeholderElement = page.getByText(/Placeholder/);
 
-  test('clicking the dropdown opens the dropdown-list, then click an option 1', async ({
-    page,
-  }) => {
-    await page.goto(componentTestPath);
-    const dropdown = page.locator('tds-dropdown');
-    const dropdownSelect = dropdown.locator('div[class="dropdown-select lg"]');
-    const button = dropdownSelect.locator('button');
-    await button.click();
-    const dropdownOptionOne = dropdown.locator('tds-dropdown-option[value="option-1"]');
-    await dropdownOptionOne.click();
-    await expect(dropdown).toHaveAttribute('value', 'option-1');
-  });
+    /* before clicking dropdownlist should not be visible, the button should be */
+    await expect(placeholderElement).toBeVisible();
+    await expect(dropdownOptionOne).not.toBeVisible();
```

Comment:
Nice that there is a check for it being hidden before as well. 
An alternative here could be .toBeHidden(). However no need as this does the same thing.

Reply:
Oh yeah its kinda the same check. feels like the not makes it easier to "read" the code that it should not be visible than toBeHidden and then to be visible but that might just be me? @timrombergjakobsson any inputs?

Reply:
I think if they both are doing the same stuff, then it does not really matter which one. Although myself I think `.toBeHidden()` is easier to read hahah. But it is just cos it maps pretty nicely with `.toBeVisible()`

Reply:
Ok have updated.